Cargo.toml

```diff
@@ -2,15 +2,15 @@
 members = [".", "migration", "client", "test-support", "cli"]
 
 [workspace.package]
-version = "0.3.21"
+version = "0.4.0"
 edition = "2021"
 license = "MPL-2.0"
 homepage = "https://divviup.org"
 repository = "https://github.com/divviup/divviup-api"
 
 [workspace.dependencies]
-divviup-client = { path = "./client", version = "0.3.21" }
-divviup-cli = { path = "./cli", version = "0.3.21" }
+divviup-client = { path = "./client", version = "0.4.0" }
+divviup-cli = { path = "./cli", version = "0.4.0" }
```

Comment:
Would `version = "0.4"` suffice? Then we wouldn't have to update the patch version all the time.

Reply:
`cargo set-version` has been taking care of that automatically so far. If we relaxed the dependency requirement to "0.4", that could open up issues in downstream projects/Cargo.lock files if we introduce new items in one crate and use them in another in the same release.